.github/workflows/pr-validation.yml

```
@@ -286,14 +317,15 @@ jobs:
-  # PHASE 1 - DOCKER BUILD (per container, always runs on PRs)
-  # All containers build in parallel regardless of changes
+  # GROUP 3 - DOCKER BUILDS (per container, always runs on PRs)
+  #   - docker-build-* jobs build images
+  #   - docker-smoke-* jobs run CI_SMOKE_MODE containers and hit /health
   # =============================================================================
 
   docker-build-bunkbot:
     name: 🐳 Docker Build - Bunkbot
     runs-on: ubuntu-latest
-    needs: [type-check, lint]
+    needs: []
```

actionlint (1.7.9)

"needs" section should not be empty

(syntax-check)